python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -8,7 +8,8 @@ import {
   CheckProfilingStatusResponse,
   getProfilingResultURL,
   launchProfiling,
-  RayletInfoResponse
+  RayletInfoResponse,
+  launchKillActor
```

Comment:
Please keep imports in sorted order. (Your editor might be able to assist with this.)

Reply:
We should have a linter for this

Reply:
I'll fix that manually and figure out the correct linter configuration for the future!